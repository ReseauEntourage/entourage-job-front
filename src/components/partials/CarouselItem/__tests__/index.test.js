import { render } from '@testing-library/react';
import React from 'react';
import { CarouselItem } from '..';
import '@testing-library/jest-dom';

describe('Carousel Item', () => {
  it('renders the Carousel item', () => {
    const defautProps = {
      index: 1,
      img: '',
      description: <div />,
    };
    const { container } = render(<CarouselItem {...defautProps} />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});
