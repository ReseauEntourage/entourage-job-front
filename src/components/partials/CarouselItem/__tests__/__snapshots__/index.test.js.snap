// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel Item renders the Carousel item 1`] = `
<div>
  <li>
    <div
      class=" uk-child-width-1-2@m uk-grid-collapse uk-grid-match uk-flex-center uk-flex-middle uk-flex-wrap uk-flex-wrap-middle"
      data-uk-grid=""
    >
      <div>
        <div
          class="uk-flex-1 uk-background-position-center-center uk-background-cover uk-height-large"
          style="background-image: url();"
        />
      </div>
      <div>
        <div
          class="uk-flex uk-flex-1 uk-flex-center uk-flex-middle uk-padding-large"
        >
          <div
            class="uk-flex uk-flex-center uk-flex-middle uk-height-medium "
          >
            <div />
          </div>
        </div>
      </div>
    </div>
  </li>
</div>
`;
